tap_liveperson/streams/messaging_queue_health.py

```diff
@@ -0,0 +1,42 @@
+from tap_liveperson.streams.base import RealtimeStream
+
+
+class MessagingQueueHealthStream(RealtimeStream):
+    API_METHOD = 'GET'
+    SERVICE_NAME = 'leDataReporting'
+    TABLE = 'messaging_queue_health'
+
+    @property
+    def api_path(self):
+        return (
+            '/operations/api/account/{}/msgqueuehealth'
+            .format(self.config.get('account_id')))
+
+    def get_params(self):
+        return {
+            "timeframe": 1440,
+            "interval": 5,
+            "skillIds": "all",
+            "v": 2
+        }
+
+    def get_stream_data(self, result):
+
+        metrics_by_skill = result.get('metricsByIntervals', [])
+
+        transformed = []
+        for records in metrics_by_skill:
+            timestamp = records['timestamp']
+            for skill, data in records['metricsData']['skillsMetrics'].items():
```

Comment:
yes it should return metrics on a skill level (so `skillsMetrics`), because we pass `"skillIds": "all",` to request parameters, but I agree that we can be on the safe side and do it as you suggested!